src/main/java/org/jboss/pnc/deliverablesanalyzer/ApplicationLifecycle.java

```diff
@@ -29,6 +30,7 @@ public class ApplicationLifecycle {
     private static final Logger LOGGER = LoggerFactory.getLogger(ApplicationLifecycle.class);
 
     public void onStart(@Observes StartupEvent event) {
+        Ansi.setEnabled(false); // Turn of color output to logs
```

Comment:
Isn't this the same as this https://github.com/project-ncl/build-finder/pull/677? Did that fix of setting the system property not work?

Reply:
this is in Deliverable analyzer, which is run as a service, so the other PR in buildfinder is not needed with this change